### ISSUE TITLE
[Bug]: Qt WebEngine PDF and Insight Tracker are now in their own Extensions category

### ISSUE THREAD

**Issue by kelteseth** [miurahr/aqtinstall#803]
### Bug description

Looks like this is intentional, see the first response https://bugreports.qt.io/browse/QTBUG-126885

![image](https://github.com/user-attachments/assets/bcb71670-5110-438a-b012-fbe14459750c)

### Expected behavior

I should be able to install them via aqt

### aqt and python version

latest from master

### Operating System

Windows

### Relevant log output

```shell
Calling 


python -m aqt list-qt windows desktop --modules 6.8.0 win64_msvc2022_64  
```

no longer list them as modules:
```
debug_info qt3d qt3d.debug_information qt5compat qt5compat.debug_information qtactiveqt qtactiveqt.debug_information qtcharts qtcharts.debug_information qtconnectivity qtconnectivity.debug_information qtdatavis3d qtdatavis3d.debug_information qtgraphs qtgraphs.debug_information qtgrpc qtgrpc.debug_information qthttpserver qthttpserver.debug_information qtimageformats qtimageformats.debug_information qtlanguageserver qtlanguageserver.debug_information qtlocation qtlocation.debug_information qtlottie qtlottie.debug_information qtmultimedia qtmultimedia.debug_information qtnetworkauth qtnetworkauth.debug_information qtpositioning qtpositioning.debug_information qtquick3d qtquick3d.debug_information qtquick3dphysics qtquick3dphysics.debug_information qtquickeffectmaker qtquickeffectmaker.debug_information qtquicktimeline qtquicktimeline.debug_information qtremoteobjects qtremoteobjects.debug_information qtscxml qtscxml.debug_information qtsensors qtsensors.debug_information qtserialbus qtserialbus.debug_information qtserialport qtserialport.debug_information qtshadertools qtshadertools.debug_information qtspeech qtspeech.debug_information qtvirtualkeyboard qtvirtualkeyboard.debug_information qtwebchannel qtwebchannel.debug_information qtwebsockets qtwebsockets.debug_information qtwebview qtwebview.debug_information
```
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tsteven4** [comment]
I acknowledge this issue is noted in https://github.com/miurahr/aqtinstall/discussions/824.  However, now that 6.8.0 has been released I would like to ask if there is a way to workaround this issue to get 6.8.0 qtwebengine installed?  Thanks.

**Comment by tsteven4** [comment]
The 6.8.0, 6.8.1 QtWebengine/Qtpdf issue with extensions was fixed in the master branch by https://github.com/miurahr/aqtinstall/pull/838, but it hasn't been released yet.

Qt Insight Tracker is only available under commercial license.

**Comment by petamas** [comment]
@tsteven4: I'm not sure whether you're aware, but although #838 fixes the installation of `qtwebengine` itself, `install-src` still does not install the `qtwebengine` sources, and there's no `qtwebengine.debug_information` module available in Qt 6.9. (Same is true for `qtpdf`, I assume.)
Tested on `aqtinstall` 3.2.1 on Windows and macOS.

**Comment by Kidev** [comment]
Yep there is an issue with extensions with commands that are not `install-qt`. You can get the source with the official installer in the meantime, I'm figuring out how to fix this nicely as all the code is already done for `install-qt`

**Comment by petamas** [comment]
@Kidev: debug info is under `install-qt`, but it cannot be installed for extensions in Qt 6.9. Or do you mean that `master` already solves this?

I'll look into the `--official-installer` option for sources, thanks!

**Comment by Kidev** [comment]
@petamas No I meant that the fixes that were done to `install-qt` to make it able to install extensions _(that's how the Qt team calls `qtpdf` and `qtwebengine` now, because those are not tied to Qt versions anymore but to chromium, [it's explained here](https://www.qt.io/blog/ensuring-product-longevity-with-qt-long-term-support) when Qt changed their LTS specs)_ were not propagated to the `install-tools` and `install-src` commands. And to be honest, I'm having a hard time finding the sources of those on the archives. 

For example, here is it for `qtwebengine` (it's called 'everywhere', not sure if its different): https://download.qt.io/online/qtsdkrepository/all_os/extensions/qtwebengine/690/src_doc_examples/extensions.qtwebengine.690.src/ 

You would assume that `qtpdf` is near, like here: https://download.qt.io/online/qtsdkrepository/all_os/extensions/qtpdf/690/src_doc_examples/
But there, there is only documentation `extensions.qtpdf.690.doc`, no sources... 

TLDR: I'm saying is that I am aware of the issue, but I have no idea yet on how to fix... I think I'm going to improve `install-qt-official` (or `--use-official-installer`) to make it less hacky to download sources from there. At least it'll make it possible to get sources that way
